dolphinscheduler-api/src/main/java/org/apache/dolphinscheduler/api/controller/ResourcesController.java

```diff
@@ -496,6 +464,7 @@ public Result createUdfFunc(@ApiIgnore @RequestAttribute(value = Constants.SESSI
                                 @RequestParam(value = "database", required = false) String database,
                                 @RequestParam(value = "description", required = false) String description,
                                 @PathVariable(value = "resourceId") int resourceId) {
+        //todo verify the sourceName
```

Comment:
Why we need `todo` here? Does this WIP?

Reply:
we  verify fewer parameters from the controller 
eg the name can't include '/'
so we need the rule which hasn`t been designed